crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -1431,12 +1431,19 @@ where
                         if let Err((hash, error)) =
                             self.try_make_sync_target_canonical(downloaded_num_hash)
                         {
-                            tracing::error!(
-                                target: "consensus::engine",
-                                "Unexpected error while making sync target canonical: {:?}, {:?}",
+                            if !matches!(
                                 error,
-                                hash
-                            )
+                                CanonicalError::BlockchainTree(
+                                    BlockchainTreeError::BlockHashNotFoundInChain { .. }
+                                )
+                            ) {
```

Comment:
there are more error variants like this that don't deserve the error! level, see BlockchainTreeError variants

this needs an evaluation what should actually be logged on error